java/k8s-addons/src/main/java/com/instaclustr/cassandra/k8s/ConcatenatedYamlConfigurationLoader.java

```diff
@@ -112,15 +113,24 @@ public Config loadConfig() throws ConfigurationException {
                     }
                 })
 
-                // only load files
+                // only load regular yaml files
                 .filter(path -> {
                     if (!Files.isRegularFile(path)) {
                         logger.warn("Configuration file \"{}\" is not a regular file and will not be loaded.", path);
                         return false;
                     }
+                    return true;
+                })
 
+                .filter(path -> {
+                    if(!path.toFile().getName().contains(".yaml")) {
```

Comment:
Instead of `Path` -> `File` -> `String`, lets use [PathMatcher](https://docs.oracle.com/javase/8/docs/api/java/nio/file/PathMatcher.html)

```
PathMatcher matcher = FileSystems.getDefault().getPathMatcher("glob:*.{yaml,yml}");

Path filename = ...;
if (matcher.matches(filename)) {
    System.out.println(filename);
}
```